packages/hardhat/hardhat.config.ts

```diff
@@ -10,6 +17,29 @@ const config: HardhatUserConfig = {
       default: 0,
     },
   },
+  networks: {
+    // order by abc
```

Comment:
Is this a ToDo?

Reply:
Thanks!! yes was an internal note, fixed here https://github.com/scaffold-eth/se-2/pull/24/commits/a4ff6938c1d9b824909ed9f407a5a200f07d2486